include/El/lattice/LLL/Left.hpp

```diff
@@ -268,14 +292,34 @@ bool Step
                 }
             }
         }
-        const Real newNorm = blas::Nrm2( m, &BBuf[k*BLDim], 1 );
+
+        if ( !colUpdated )
+        {
+            break;
+        }
+
+        colUpdated = false;
+
+        Real newNorm = lll::Norm2<Z,F>(B, k);
+        auto rCol  = QR( ALL, IR(k) );
+        Real rNorm = El::FrobeniusNorm(rCol);
+
+        if (Abs(newNorm - rNorm)/newNorm >= thresh)
```

Comment:
This is pedantic, but for the sake of consistency, I would recommend using `if( condition )` rather than `if (condition)` since the former is what the rest of the library uses (and there is a slippery slope).